c_glib/gandiva-glib/node.h

```diff
@@ -26,43 +26,52 @@
 G_BEGIN_DECLS
 
 #define GGANDIVA_TYPE_NODE (ggandiva_node_get_type())
+GGANDIVA_AVAILABLE_IN_0_12;
```

Comment:
I just noticed I accidentally added semicolons at the end of the macros in this header, which causes a problem when the macros get replaced with something. I can fix this as part of #41599